MagnetScannerIOS/ScannerGeolocation.swift

```diff
@@ -10,88 +10,36 @@ import CoreLocation
 import SwiftyJSON
 
 class ScannerGeolocation: NSObject, CLLocationManagerDelegate, Scanner {
-  let locationManager: CLLocationManager = CLLocationManager()
+  var locationResolver: OneShotLocation = OneShotLocation()
```

Comment:
The way of getting location in ios means you will receive plenty of callbacks when you ask for a location.
OneShotLocation means this class will give you one single location, one call, one callback :)

But I'm open to rename anything :P, suggestions welcome.